supabase/migrations/20240315103859_create_packing_slot_delete_and_update_function.sql

```diff
@@ -0,0 +1,42 @@
+set check_function_bodies = off;
+
+CREATE OR REPLACE FUNCTION public.packing_slot_delete_and_update(rowid uuid)
```

Comment:
If we decide to remove the functionality to delete packing slots, this shouldn't be needed